backend/src/apiserver/resource/resource_converter.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Kubeflow Authors
+// Copyright 2018-2022 The Kubeflow Authors
```

Comment:
I'm not sure if it's a good idea to rename the file `model_converter.go` to `resource_converter.go`. Here `model` means that the data is converted to model types. Symmetrically we have `api_converter.go` in `server` folder.

Reply:
One of the goals of the refactoring of the api server is to remove the dependency on the auto-generated go-packages. To achieve that, we convert API -> model and model -> API at the very first layer - `server` package. The underlying layers work with the internal representation types that sit in `model`. These types are DB-writable and we use `gorm` annotation to define the schema there too.